nearcore/src/state_sync.rs

```diff
@@ -59,10 +68,11 @@ pub fn spawn_state_sync_dump(
         runtime.num_shards(&epoch_id)
     }?;
 
+    let chain_id = client_config.chain_id.clone();
+    let keep_running = Arc::new(AtomicBool::new(true));
     // Start a thread for each shard.
     let handles = (0..num_shards as usize)
```

Comment:
not a problem right now, but in a future world where we have 1000 shards or something, one `actix_rt::Arbiter::new()` per shard is probably not what we want

Reply:
Acknowledged